docs/source/geometry.bbox_v2.rst

```diff
@@ -0,0 +1,11 @@
+kornia.geometry.bbox
```

Comment:
Which is the command to generate docs locally? I don't know it and I couldn't check if it appear in kornia docs.

Reply:
from project root you can use `make build-docs`